--- a/nall/path.cpp
+++ b/nall/path.cpp
@@ -143,7 +143,7 @@ NALL_HEADER_INLINE auto sharedData() -> string {
   #elif defined(PLATFORM_MACOS)
   string result = "/Library/Application Support/";
   #else
-  string result = "/usr/share/";
+  string result = "/app/share/";
   #endif
   if(!result) result = ".";
   if(!result.endsWith("/")) result.append("/");
@@ -152,7 +152,7 @@ NALL_HEADER_INLINE auto sharedData() -> string {
 
 #if defined(PLATFORM_LINUX) || defined(PLATFORM_BSD)
 NALL_HEADER_INLINE auto prefixSharedData() -> string {
-  string result = program().append("../share/");
+  string result = program().append("/app/share/");
   return result;
 }

